Whisky/Models/Bottle.swift

```diff
@@ -115,6 +115,16 @@ extension Bottle {
         // Apply blocklist
         programs = programs.filter { !settings.blocklist.contains($0.url) }
 
+        // Apply pinned list as programs, if not already included
+        settings.pins.forEach { pin in
+            if let pinnedUrl = pin.url {
+                let program = Program(name: pin.name, url: pinnedUrl, bottle: self)
```

Comment:
We probably want to check if this program even exists anymore, maybe we can make Program a fallible init or something. Currently moving/renaming a manually pinned file doesn't remove it from the menu which is kind of confusing for end user.

Reply:
I put this in originally because I needed to configure an environment variable on one of my manually pinned programs, and couldn't since it wasn't in the `programs` array, so it wasn't as fleshed out as it could have been. The concept of configuring a manually pinned program could probably stand to be reviewed more in-depth anyway. If you want to continue with this type of implementation, I can add detection logic into the `Program` init.